app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/SearchResultsActivity.java

```diff
@@ -1,10 +1,64 @@
 package edu.grinnell.appdev.grinnelldirectory.Activity;
 
+import android.os.Bundle;
+//import android.support.v7.app.AlertController;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+import edu.grinnell.appdev.grinnelldirectory.R;
+import edu.grinnell.appdev.grinnelldirectory.SearchResultsAdapter;
+
 /**
  * Created by ritikaagarwal on 2/19/17.
  */
 
-public class SearchResultsActivity {
+public class SearchResultsActivity extends AppCompatActivity {
+    private List<Person> mPeople;
+
+    public void populatePeopleList() {
```

Comment:
I like that you test the method here but this test code should not being in the codebase ready to be merged into `develop`. You should just add a screenshot of what shows when you run the app.